examples/examples/z_bytes.rs

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright (c) 2024 ZettaScale Technology
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+//
+// Contributors:
+//   ZettaScale Zenoh Team, <zenoh@zettascale.tech>
+//
+
+use std::{borrow::Cow, collections::HashMap};
+
+use zenoh::bytes::ZBytes;
+
+fn main() {
+    // Numeric: u8, u16, u32, u128, usize, i8, i16, i32, i128, isize, f32, f64
```

Comment:
The corresponding default encoding should be added in the example.

Reply:
Do you mean we should have the actual put and sub here? ZBytes doesn't include the encoding information unless we use Sample.

Reply:
I mean something like:

```rust
let input = 1234_u32;
let payload = ZBytes::from(input);
let output: u32 = payload.deserialize().unwrap();
assert_eq!(input, output); 
// Corresponding encoding
// let encoding = Encoding::ZENOH_INT32;
```

Reply:
Done